airbyte-integrations/connectors/source-postgres/integration_tests/incremental_configured_catalog_template.json

```diff
@@ -2,23 +2,22 @@
   "streams": [
     {
       "stream": {
-        "name": "id_and_name",
+        "name": "id_and_name_hook",
```

Comment:
this `_hook` still sounds weird... can we find a better name.. maybe `_cat` or something more expressive on the usage or meaning.

Reply:
updated!